src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
```

Comment:
So I'm not sure why its not always a value?  Are bothered by the subtle implications between objects/value types?  I don't think people will stumbled on that since it works the same for both today no?